DOL.WHD.Section14c.Web/src/modules/components/mainTopNavControl/mainTopNavControlDirective.js

```diff
@@ -1,17 +1,5 @@
-'use strict';
+import { mainTopNav } from './mainTopNav.component';
 
 module.exports = function(ngModule) {
-  ngModule.directive('mainTopNavControl', function() {
-    'use strict';
-
-    return {
-      restrict: 'EA',
-      template: require('./mainTopNavControlTemplate.html'),
-      controller: 'mainTopNavControlController',
-      scope: {
-        admin: '='
-      },
-      controllerAs: 'vm'
-    };
-  });
+  ngModule.component('mainTopNavControl', mainTopNav);
 };
```

Comment:
This refactors the old AngularJS directive into a component and uses the directive metadata from the new `main-top-nav` directive, so they'll be the same.

Reply:
I'm curious if the directive metadata should live here, since this is the "real" piece of functionality?  And then it could be `import`ed in the wrapper component?  As-is, the "old" functionality has a dependency on the new wrapper, which seems backwards.  🙂

Reply:
good point, will change!

Reply:
updated, tysir :)